test/Constraints/casts.swift

```diff
@@ -741,3 +741,17 @@ do {
     assert(Set(b.map { $0 as A }) == Set(a)) // Ok
   }
 }
+
+// https://github.com/apple/swift/issues/68825
+do {
+  func x(a: Any) {
+    _ = a is a
+    // expected-error@-1 {{type-casting operator expects a type on its right-hand side (got: parameter 'a')}}
+    // expected-note@-3 {{'a' declared here}}
```

Comment:
Please attach `// expected-note {{'a' declared here}}` to the declaration of `a`, it makes it a lot easier to read also you can use `// expected-note <Number-of-Occurances> {{<message>}}` to indicate that the same note expected to appear multiple times i.e. in this case it would be `// expected-note 2 {{'a' declared here}}` on line 747.

Same applies to the rest of the modified test-cases.

Reply:
Thanks, the code looks much cleaner. Done.